client/boot/common.js

```diff
@@ -217,7 +216,7 @@ const utils = () => {
 	accessibleFocus();
 
 	// Configure app element that React Modal will aria-hide when modal is open
-	Modal.setAppElement( document.getElementById( 'wpcom' ) );
+	Modal.setAppElement( getRootDomElement() );
```

Comment:
If you kept the `document.getElementById` call here, you could avoid exporting `getRootDomElement`. That's what I would prefer, but it's not very important 🙂

Reply:
Yeah, I've also thought about whether enforcing [DRY](https://en.wikipedia.org/wiki/Don%27t_repeat_yourself) to this level is worth it or not here. I'd like to emphasize the idea that `Modal.setAppElement` here should be attached to the same DOM element like the React root does. However, that comes with the price of having to export that function. If there are more cases like `Modal.setAppElement` here, it might worth the cost, but we likely won't see them coming 🤔 
Since it's not too important, I'd like to keep it this way for now and see how it goes :)